generate_config.sh

```diff
@@ -163,6 +193,17 @@ git checkout -f $git_branch
 
 [ ! -f ./data/conf/rspamd/override.d/worker-controller-password.inc ] && echo '# Placeholder' > ./data/conf/rspamd/override.d/worker-controller-password.inc
 
+if [[ "${CONTAINER_ENGINE}" == "podman" ]]; then
+    MAILCOW_HTTP_BIND="127.0.0.1"
+    MAILCOW_HTTPS_BIND="127.0.0.1"
```

Comment:
Is this really needed? This will prevent mailcows UI being reachable over IPv6 by default and any manual configurations.

IPv6 will only be available if the user sets the IPv6 according to this:
https://docs.mailcow.email/post_installation/firststeps-ip_bindings/#ipv6-binding

Reply:
I just retried things and apparently this indeed isn't needed (I think i added this because compose was complaining but that might have been compose v1 still). Also setting the bind address for MySQL is not needed. I have removed/reverted both changes.